dpnp/tests/test_usm_type.py

```diff
@@ -1197,26 +1120,26 @@ def test_where(usm_type):
         "(1, 0, 0)",
     ],
 )
-@pytest.mark.parametrize("usm_type", list_of_usm_types, ids=list_of_usm_types)
+@pytest.mark.parametrize("usm_type", list_of_usm_types)
 def test_eigenvalue(func, shape, usm_type):
```

Comment:
Would it make sense to group linalg functions by `TestLinalg` class? (like it is done for fft functions)